src/Workspace/index.jsx

```diff
@@ -1,17 +1,53 @@
 import React from 'react';
-import { workspaceUrl } from '../localconf';
+import { workspaceUrl, wtsPath } from '../localconf';
 import './Workspace.less';
+import { fetchWithCreds } from '../actions';
+import Spinner from '../components/Spinner';
 
 class Workspace extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      connectedStatus : false,
+    }
+  }
+
+  connected() {
+    console.log("connected");
+    this.setState({connectedStatus : true});
+  }
+
+  componentDidMount() {
+    fetchWithCreds({
+      path: `${wtsPath}connected`,
+      method: 'GET',
+    })
+    .then(
+      ({status}) => {
+        if (status !== 200) {
+          window.location.href = `${wtsPath}/authorization_url?redirect=/workspace`;
```

Comment:
I think the redirect can be a variable? like `windows.location.href` or something, so the dev.html is captured